src/test/java/com/github/fishio/power_ups/TestPuSuperSpeed.java

```diff
@@ -0,0 +1,87 @@
+package com.github.fishio.power_ups;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.mockito.Mockito;
+
+import com.github.fishio.PlayerFish;
+import com.github.fishio.PlayingField;
+
+/**
+ * Tests the PuSuperSpeed class.
+ */
+public class TestPuSuperSpeed extends TestDurationPowerUp {
+
+	private PuSuperSpeed pu;
+	private PlayingField pf;
+	
+	/**
+	 * Creates a new PoFreeze object to test and sets
+	 * the PlayingField pf field.
+	 */
+	@Before
+	public void setUp() {
+		this.pf = Mockito.mock(PlayingField.class);
+		this.pu = Mockito.spy(new PuSuperSpeed(null, pf));
+		
+		//To prevent NullPointerExceptions, mocking the target of the PowerUp.
+		this.pu.setTarget(Mockito.mock(PlayerFish.class));
+	}
+	
+	@Override
+	public DurationPowerUp getDurationPowerUp() {
+		return pu;
+	}
+
+	@Override
+	public void testGetDuration() {
+		assertEquals(10, pu.getDuration());
+	}
+
+	@Override
+	public void testStartEffect() {
+		PlayerFish pf = pu.getTarget();
+		double oldAcceleration = pf.getAcceleration();
+		double oldMaxSpeed = pf.getMaxSpeed();
+		
+		pu.startEffect();
+		
+		assertEquals(oldAcceleration *  PuSuperSpeed.ACCELERATION_FACTOR, 
+				pf.getAcceleration(), 0.0000001D);
```

Comment:
You could make the 0.0000001D into a private static final double DELTA = 0.0000001D;

<naming,layout